docs/runbooks/CDIDataImportCronOutdated.md

```diff
@@ -69,4 +69,13 @@ $ kubectl get dataimportcron cron-test -o yaml | grep -B 5 storageClassName
         storageClassName: rook-ceph-block
 ```
 
+### Disconnected Environment
+
+For a cluster installed in a disconnected environment, the default golden images can't be polled because they are out of reach, thus the alert will be fired.  
+In that case, it is encouraged to disable the feature gate `enableCommonBootImageImport` to opt-out from automatic updates of the common data import cron templates and refrain from this alert being fired.  
```

Comment:
I'd add a link to HCO docs about this.

Reply:
added.

Reply:
Dummy question - you are limited to use a local registry, but why must you disable the feature gate?

Reply:
Disabling the feature gate in a disconnected environment is not a "must", it is just encouraged, to avoid the alert being fired, and to avoid the cluster being uselessly attempting to reach the default sources, since internet connection is out of reach for these environments.